docs/frameworks/dapr.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Use Dapr with .NET Aspire
+description: Learn how to use Dapr with .NET Aspire
+ms.date: 05/14/2024
+ms.topic: overview
+---
+
+# Use Dapr with .NET Aspire
+
+[Distributed Application Runtime (Dapr)](https://docs.dapr.io/) offers developer APIs that run as a sidecar process and abstract away the common complexities of the underlying cloud platform. Dapr and .NET Aspire work together to improve your local development experience. By using Dapr with .NET Aspire, you can focus on writing and implementing .NET-based distributed applications instead of spending extra time with local onboarding.  
+
+In this guide, you'll learn how to take advantage of Dapr's abstraction and .NET Aspire's opinionated configuration of cloud technologies to build simple, portable, resilient, and secured microservices at-scale on Azure.
+
+## Prerequisites
+
+- TODO: Add prerequisites
```

Comment:
I'd suggest making Dapr 1.13 (the latest release) a prerequisite (see: https://docs.dapr.io/getting-started/install-dapr-cli/), and also mention that the Dapr runtime should be initialized on the machine (see: https://docs.dapr.io/getting-started/install-dapr-selfhost/).